From 2d9301320af9dfdc3fb571da916732db916ce5db Mon Sep 17 00:00:00 2001
From: Agriya Khetarpal <74401230+agriyakhetarpal@users.noreply.github.com>
Date: Sun, 24 Mar 2024 04:53:35 +0530
Subject: [PATCH] use-temporary-directory-to-store-eggs

This patch sets the egg_base in setup.cfg to store the eggs in a
temporary directory when building the source distribution. This
is useful when avoiding the inclusion of the egg.info files in
the tar.gz file generated by the build frontend.

---
 setup.cfg | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/setup.cfg b/setup.cfg
index 7752624..041eb2b 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -9,3 +9,6 @@ console_scripts =
 license_files =
     LICENSE
     LICENSE-hugo.txt
+
+[egg_info]
+egg_base = /tmp
--
2.44.0
